examples/tf/ppo_pendulum.py

```diff
@@ -0,0 +1,55 @@
+"""
+This is an example to train the HalfCheetah-v2 task with PPO algorithm.
+
+Here it creates HalfCheetah using gym. And uses a PPO with 1M
+steps.
+
+Results:
+    AverageReturn: 526.092
+    RiseTime: itr 488
+"""
+import gym
+
+from garage.baselines import LinearFeatureBaseline
+from garage.envs import normalize
+from garage.misc.instrument import run_experiment
+from garage.tf.algos import PPO
+from garage.tf.envs import TfEnv
+from garage.tf.policies import GaussianMLPPolicy
+
+
+def run_task(*_):
+    """
+    Wrap PPO training task in the run_task function.
+
+    :param _:
+    :return:
+    """
+    env = TfEnv(normalize(gym.make("InvertedDoublePendulum-v2")))
```

Comment:
You mention using HalfCheetah in your description, but use InvertedDoublePendulum here?